Block/System/Config/TbkButton.php

```diff
@@ -24,8 +24,13 @@ public function __construct(
         $healthcheck = new HealthCheck($config);
         $datos_hc = json_decode($healthcheck->printFullResume());
 
-        $log = new LogHandler();
-        $resume = $log->getResume();
+        $logger = new PluginLogger();
+        $logInfo = $logger->getInfo();
+        $logDetail = [];
+
+        if (isset($logInfo['last'])) {
```

Comment:
this condition is always `true`, since the `getInfo` method by default returns `''`, which is considered as `true` by isset()